code/game/objects/effects/temporary_visuals/projectile/impact.dm

```diff
@@ -0,0 +1,52 @@
+/obj/effect/projectile/impact
```

Comment:
Can you add a comment above this explaining what impact projectiles are / how they are used as part of hitscan?

Reply:
doen